service/service_account.go

```diff
@@ -70,11 +69,11 @@ func (s AccountService) AccountBalance(
 	}
 
 	var balances []*types.Amount
-	balances = append(balances, mapper.AvaxAmount(avaxBalance))
 	for _, currency := range req.Currencies {
 		value, ok := currency.Metadata[mapper.ContractAddressMetadata]
 		if !ok {
-			if currency.Decimals == AvaxDecimals && strings.ToLower(currency.Symbol) == AvaxSymbolLower {
+			if types.Hash(currency) == types.Hash(&types.Currency{Symbol: AvaxSymbol, Decimals: AvaxDecimals}) {
+				balances = append(balances, mapper.AvaxAmount(avaxBalance))
```

Comment:
You also need to add this if `req.Currencies` is `len==0`

Reply:
So exactly the way it was before?